playbooks/adhoc/create_pv/create_pv.yaml

```diff
@@ -20,7 +20,7 @@
   pre_tasks:
   - fail:
       msg: "This playbook requires {{item}} to be set."
-    when: "{{ item }} is not defined or {{ item }} == ''"
+    when: {{ item }} is not defined or {{ item }} == ''
```

Comment:
Fixed in https://github.com/openshift/openshift-ansible/pull/4527/commits/8247c7a5bad8bb34148071fbc820b21bb09d82e8